src/Catty/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -766,12 +766,12 @@
 /* No comment provided by engineer. */
 "Go to front" = "Go to front";
 
-/* paint */
-"Green" = "Green";
-
 /* No comment provided by engineer. */
 "green" = "green";
 
+/* paint */
+"Green" = "Green";
+
```

Comment:
It probably doesn't matter, but what are these color-string changes to the same value as deleted above for?